tests/models/layoutlmv3/test_tokenization_layoutlmv3.py

```diff
@@ -1615,7 +1616,8 @@ def test_batch_encode_dynamic_overflowing(self):
                     returned_tensor = "jax"
 
                 # Single example
-                words, boxes = self.get_words_and_boxes()
+                words = ["HuggingFace", "is", "solving", "NLP", "one", "commit", "at", "a", "time"]
+                boxes = [[i,i,i,i] for i in range(len(words))]
```

Comment:
There was an error ([here](https://app.circleci.com/pipelines/github/huggingface/transformers/39226/workflows/9e2943f6-8cf8-4a3f-86da-f74fb218e52a/jobs/439608)) because the examples chosen were not long enough to be truncated